src/containers/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ErrorPage from "../../components/ErrorPage/ErrorPage.js";
+
+export default class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError(error) {
```

Comment:
It would be nice if you also add componentDidCatch with logging, it can be useful for debugging.

Reply:
Added componentDidCatch with logging to the console, but it will be useful in the future to log errors to some error reporting service.